changelogs/fragments/1213-hashi_vault-jwt-auth-support.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - hashi_vault - add support for JWT authentication (https://github.com/ansible-collections/community.general/pull/1213).
```

```suggestion
  - hashi_vault lookup plugin - add support for JWT authentication (https://github.com/ansible-collections/community.general/pull/1213).
```
We only leave that away for modules :)